hydradx/notebooks/Omnipool/LP_returns_analysis2.ipynb

```diff
@@ -0,0 +1,437 @@
+      "text/plain": [
+       "<Figure size 1440x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "price_path = []\n",
+    "modifier_list = [-0.5, -0.25, 0, 0.25, 0.5]\n",
+    "colors = pu.color_gradient(length=5, color1=(255, 0, 0), color2 = (0, 255, 0))\n",
+    "initial_dot_price = price_list[0]['DOT']\n",
+    "\n",
+    "plt.figure(figsize=(20, 5))\n",
+    "plt.title(\"Price paths from optimistic to pessimistic\")\n",
+    "\n",
+    "for i, price_shift in enumerate(modifier_list):\n",
+    "    price_shift_per_block = price_shift / time_steps\n",
+    "    price_path.append([\n",
+    "        {('INTR' if tkn == 'DOT' else tkn): (\n",
```

Comment:
I don't understand what happens do the DOT price here? Are we somehow starting with DOT in the Omnipool and not INTR and then switching?

Reply:
To clarify.... isn't this going to result in the key 'INTR' getting written to twice in this dict (and I think overwritten the 2nd time) but the key 'DOT' getting written to 0 times?

Reply:
INTR should only get written to once, since it is not present in the initial price list. I'm just using DOT price as a stand-in for INTR, but if there is data available for INTR I should use that instead.